### PR TITLE
clean up

### PR DESCRIPTION
**Pull request by kimamov**

### CODE REVIEW

File: src/api/useFetch.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useState, useEffect } from 'react';
 
-export const BASE_PATH='https://jobs-api.communityredi.school';
-//export const BASE_PATH='http://localhost:8000';
+//export const BASE_PATH='https://jobs-api.communityredi.school';
```
Comment (lopis) [inline src/api/useFetch.tsx]: I think you forgot to revert this, right?

Comment (kimamov) [reply src/api/useFetch.tsx]: Ohh ya sry

**Review by lopis — CHANGES_REQUESTED** [review]
LGTM but can you confirm if that URL change was a mistake?

**Comment by kimamov** [comment]
> LGTM but can you confirm if that URL change was a mistake?

should be fixed now :)

**Review by lopis — APPROVED** [review]

**Pull request merged**